### PR TITLE
Refs #509 -- Added a tutorial to introduce how to cross-space CRC20 token to eSpace 

### PR DESCRIPTION
**Pull request by jackleeio**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://doc.confluxnetwork.org/docs/general/CONTRIBUTING#create-a-pull-request).
- [x] A relating issue is created: #509 (<- fill the issue number here)
- [x] **This is NOT a PR to submit translation**

- - -
This change is [Reviewable](https://reviewable.io/reviews/Conflux-Chain/conflux-documentation/592)

### CODE REVIEW

**Review by darwintree — CHANGES_REQUESTED** [review]
I think current version needs refactoring. As previously discussed, this article mainly serve as a **tutorial** for how to register the token and enable core space erc20 token bridging to espace, which means, the internal implementation is not important and will interfere the article's readability. The contract imlementation illustration can be put into the planned article on `CrossSpaceCall` explanation

How to modify:

1. (use the cofluxscan contract view) just focus on the operation with simple illustration. Then this article can be moved into the tutorial folder. Or
2. directly providing js/python scripts on the operation.

The difference depends on the target audience

File: docs/core/build/cross-space/crc20-cross-space.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 1
+title: CRC20 Tokens Cross Space
+displayed_sidebar: coreSidebar
+---
+
+This tutorial guides developers on transferring any CRC20 token from Core Space to eSpace, using [FansCoin (FC)](https://confluxscan.io/token/cfx:achc8nxj7r451c223m18w2dwjnmhkd6rxawrvkvsy2) as an example. The goal is to transfer a CRC20 token (A) issued on coreSpace to become an ERC20 token (eA) in eSpace. Currently, if there is no corresponding token in eSpace, it's not possible to use the official [cross-space bridge](https://confluxhub.io/espace-bridge/cross-space) directly.
```
Comment (darwintree) [inline docs/core/build/cross-space/crc20-cross-space.md:7]: Using FC as example seems improper here as the main content of this article is now how to enable token transfer.

**Review by darwintree — CHANGES_REQUESTED** [review]

**Comment by jackleeio** [comment]
Are there any other issues? @darwintree

**Comment by darwintree** [comment]
> Are there any other issues? @darwintree

Apologies for the delay in reviewing the recent articles. Our current focus is on addressing issues related to the upcoming Conflux hard fork. We estimate that the most urgent tasks will be completed by the end of next week, though it could take longer. We appreciate your patience if we are unable to review the recent pull requests in the coming days.

**Review by darwintree — CHANGES_REQUESTED** [review]
This article is moved to the tutorial folder, and should be considered as a normal-user-oriented article.

1. For most users, the step `sing the Official Cross-Space Bridge` is enough. It is recommended to place it to the first place.
2. If the token bridging is not applicable (not registered before), then reader can then check the currrent step 1 and 2 to register token and create token mapping
3. Recommeded to remove current step 4 and 5 as currently this is enough for users, for example, if he or she would like to add liquidity to swappi. However, it is still acceptable if you would like to add relating content, but current version is not appropriate. For developers, it is a more reliable and authentic way to show how to modify the parameter in https://github.com/Conflux-Chain/conflux-dapps to deploy a fork version of the frontend

**Comment by jackleeio** [comment]
This tutorial is clearly intended for developers. It is not suitable for most ordinary users, as their needs are completely different from the topic of this article. It’s unlikely that non-developers will be issuing a token and crossing space to eSpace.

For your first point, I recommend creating a simpler tutorial based on the Step 3 extension, rather than changing the existing structure.

Regarding the third point, as I mentioned before, this is an issue developers will likely encounter when their project issues a token and needs to cross space to eSpace.

File: docs/core/tutorials/crc20-cross-space.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+![add logo](./imgs/cross-space/add-logo.jpg)
+
+### Step 5: Displaying on Cross-Space Bridge
+
+To be displayed on the official cross-chain bridge, modify the `native_token_list_mainnet.json` file and submit a pull request to the [conflux-evm-bridge](https://github.com/Conflux-Chain/conflux-evm-bridge) repository, requesting to add your token to the default display.
+
+![default display](./imgs/cross-space/default-display.png)
+
+You can also visit the community-deployed version at [community cross-space bridge](https://bridge.fanscoin.org), or submit a PR at this [repository](https://github.com/ConfluxDAO/conflux-evm-bridge/) to directly display your token on the community cross-space bridge.
```
Comment (darwintree) [inline docs/core/tutorials/crc20-cross-space.md:63]:
```suggestion
The frontend code of ConfluxHub is hosted at [conflux-dapps](https://github.com/Conflux-Chain/conflux-dapps). You can also customize your own version of token list as you wish by modifying the parameter [innerTokenListUrl(in dapps/cross-space/src/components/TokenList/tokenListStore.ts)](https://github.com/Conflux-Chain/conflux-dapps/blob/dev/dapps/cross-space/src/components/TokenList/tokenListStore.ts) and then deploy it.
```

**Review by darwintree — CHANGES_REQUESTED** [review]

**Review by darwintree — APPROVED** [review]

**Pull request merged**